src/Sidebar.js

```diff
@@ -51,7 +51,7 @@ const Label = styled.span`
 `;
 
 const context = createContext({
-  currentPath: '/',
+  currentPath: "/"
```

Comment:
Trailing commas are a good habit, they should be kept

Reply:
Sorry about that, it looks like its a configuration from my editor